.github/actions/cypress-e2e-testing/action.yml

```diff
@@ -55,16 +55,25 @@ runs:
         retry_on: error
         command: cd ${{ inputs.module }}/tests/e2e && pnpm cypress install --force
 
+    # - name: Cypress end-to-end testing
+    #   uses: cypress-io/github-action@v6
+    #   env:
+    #     CYPRESS_VERIFY_TIMEOUT: "60000"
+    #     CYPRESS_WEB_IMAGE_OS: ${{ inputs.web_image_os }}
+    #     CYPRESS_WEB_IMAGE_VERSION: ${{ inputs.web_image_version }}
+    #     CYPRESS_OPENID_IMAGE_VERSION: ${{ inputs.openid_image_version }}
+    #   with:
+    #     browser: chrome
+    #     install: false
+    #     working-directory: ${{ inputs.module }}/tests/e2e
+    #     spec: cypress/e2e/**/${{ inputs.feature_file_path }}
+    #     publish-summary: false
+
     - name: Cypress end-to-end testing
-      uses: cypress-io/github-action@v6
+      run: |
+        pnpm run cypress:run --spec cypress/e2e/**/${{ inputs.feature_file_path }}
```

Comment:
you should use this command with cypress action : https://github.com/cypress-io/github-action#custom-test-command